.gitignore

```diff
@@ -10,6 +10,12 @@ tools/claat
 gcloud-client-secret.json
 package-lock.json
 
+## IDE related files
```

Comment:
Can you remove this file from the PR please?

You can add IDE related files to your personal `.gitignore` file, and `index.yaml` files are not generated anywhere, there are some `_index.yaml`, but those are already handled.

Reply:
Sure.
I guess [that could explain](https://cloud.google.com/appengine/docs/standard/python/config/indexconfig#creating_indexes_using_the_development_server) why I had autogenerated index.yaml upon the first launch on my machine.

Done.